.prettierrc.yaml

```diff
@@ -0,0 +1,4 @@
+trailingComma: 'all'
+tabWidth: 2
+semi: true
+singleQuote: true
```

Comment:
sorry if it is a silly question, but why is this file here? 
I thought we had removed prettier altogether...